### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by NullableInt**
This pull request aims to solve the issue of packages we need in docker not always being installed even though a `.apk` file is defined.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,22 +29,31 @@ like to host it locally on your computer, you need to do the following:
...
-   * `supperchong.pretty-json` to format selected JSON snippets in code
-     examples.
-   * `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
-8. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
-   by adding `"editor.rulers": [80]` to its configuration.
+
+### Docker
+
+1. Install [Docker][docker], using Linux containers.
+   Make sure virtualization is enabled on your machine.
```
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554&bid=17590198)

File: README.md [hunk 2]
```diff
@@ -61,12 +73,13 @@ This website is available as open source under the terms of the
...
-[license]: https://opensource.org/licenses/MIT
-[liquid]: https://jekyllrb.com/docs/liquid/
-[repo]: https://github.com/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/
-[ruby]: https://www.ruby-lang.org/en/
-[swedbankpay]: https://swedbankpay.github.io/swedbank-pay-design-guide-jekyll-theme/
-[opengraph-image]: https://repository-images.githubusercontent.com/209270355/36818080-53ee-11ea-896c-082addb851a6
+[docker]: https://www.docker.com/
+[gems]: <https://rubygems.org/>
+[ghp]: <https://pages.github.com/>
+[jekyll]: <https://jekyllrb.com/>
```
Comment (payex-dev) [inline README.md:79]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md [hunk 3]
```diff
@@ -26,22 +29,31 @@ like to host it locally on your computer, you need to do the following:
...
-     examples.
-   * `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
-8. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
-   by adding `"editor.rulers": [80]` to its configuration.
+
+### Docker
+
+1. Install [Docker][docker], using Linux containers.
+   Make sure virtualization is enabled on your machine.
+2. Open a console window at the root of the repository and run
```
Comment (payex-dev) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md [hunk 4]
```diff
@@ -26,22 +29,31 @@ like to host it locally on your computer, you need to do the following:
...
+* `bpruitt-goddard.mermaid-markdown-syntax-highlighting`, to give syntax
+  highlighting to Mermaid diagrams in Markdown files.
+* `yzhang.markdown-all-in-one`, to enable a plethora of Markdown features,
+  most importantly formatting of Markdown tables with VS Code's built-in
+  format functionality.
+* `stkb.rewrap`, to make line-breaking text at 80 characters easier.
+* `supperchong.pretty-json` to format selected JSON snippets in code
+  examples.
+* `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
+* [Set up a ruler at 80 characters][vsc-ruler] by
```
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md [hunk 5]
```diff
@@ -61,12 +73,13 @@ This website is available as open source under the terms of the
...
-[jekyll]: https://jekyllrb.com/
-[license]: https://opensource.org/licenses/MIT
-[liquid]: https://jekyllrb.com/docs/liquid/
-[repo]: https://github.com/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/
-[ruby]: https://www.ruby-lang.org/en/
-[swedbankpay]: https://swedbankpay.github.io/swedbank-pay-design-guide-jekyll-theme/
-[opengraph-image]: https://repository-images.githubusercontent.com/209270355/36818080-53ee-11ea-896c-082addb851a6
+[docker]: https://www.docker.com/
+[gems]: <https://rubygems.org/>
+[ghp]: <https://pages.github.com/>
```
Comment (payex-dev) [inline README.md:78]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md [hunk 6]
```diff
@@ -26,22 +29,31 @@ like to host it locally on your computer, you need to do the following:
...
-   * `stkb.rewrap`, to make line-breaking text at 80 characters easier.
-   * `supperchong.pretty-json` to format selected JSON snippets in code
-     examples.
-   * `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
-8. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
-   by adding `"editor.rulers": [80]` to its configuration.
+
+### Docker
+
+1. Install [Docker][docker], using Linux containers.
```
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 4 above)
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 1 above)
Comment (NullableInt) [reply README.md:36]: Why would you want a newline after list item?

File: README.md (hunk 3 above)
Comment (NullableInt) [reply README.md:37]: But it's correctly indented.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554&bid=17590198)

File: README.md (hunk 3 above)
Comment (payex-dev) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 4 above)
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554&bid=17590198)

File: README.md (hunk 3 above)
Comment (payex-dev) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 4 above)
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554&bid=17590198)

File: README.md (hunk 3 above)
Comment (payex-dev) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 4 above)
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554&bid=17590198)

File: README.md (hunk 3 above)
Comment (payex-dev) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 4 above)
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554&bid=17590198)

File: README.md (hunk 3 above)
Comment (payex-dev) [inline README.md:37]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 4 above)
Comment (payex-dev) [inline README.md:55]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

**Comment by NullableInt** [comment]
> 
> 
> This seems to work wonderfully! ❤️ But is it possible to get rid of the following warning?
> 
> > Warning: the running version of Bundler (2.0.2) is older than the version that created the lockfile (2.1.4). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.

Of course it is! And it's already done

**Pull request merged**